docs/UserGuide.md

```diff
@@ -222,18 +229,23 @@ Examples:
     - `reviewed -d` shows a list of all flashcards sorted according to review frequency in descending order
     - `success -a` shows a list of all flashcards sorted according to success rate in ascending order
     - `success -d` shows a list of all flashcards sorted according to success rate in descending order
+
+<div markdown="span" class="alert alert-primary">:memo: Note: The review and success frequency of a flashcard is only affected by quiz mode.
```

Comment:
The review and success frequencies of a flashcard are only affected by quiz mode.